returnn/tf/layers/basic.py

```diff
@@ -7320,7 +7321,7 @@ def __init__(self, factor, axis, out_dim=None, kind="nn", fill_value=None, fill_
         out_dyn_size = tf.reduce_sum(tf.cast(tf.logical_and(mask, orig_mask), tf.int32), axis=1)
     if axis != 1:
       perm = [0] + remaining_axes
-      perm.insert(axis, 1)
+      perm.insert(1, axis)
```

Comment:
Right.

But then we can simplify this directly to:
```
x = tf.transpose(x, [0, axis] + remaining_axes)
```

Reply:
But why actually? This `transpose` should revert the previous transpose above. The old code looks correct to me. Can you give an example?

Reply:
You can see an example in the test case that I added. The input is `(B,F,T)` and `T` is resized.

Reply:
Ah right, but the wrong part here is `remaining_axes` and not this `insert`.
But anyway, see my suggestion, then we don't need the `transpose` at all.